flink-connectors/flink-connector-kafka-0.10/pom.xml

```diff
@@ -86,40 +86,8 @@ under the License.
 			<optional>true</optional>
 		</dependency>
 
-		<dependency>
-			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-avro</artifactId>
-			<version>${project.version}</version>
-			<!-- Projects depending on this project, won't depend on flink-avro. -->
-			<optional>true</optional>
-		</dependency>
-
-		<dependency>
-			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-json</artifactId>
-			<version>${project.version}</version>
-			<!-- Projects depending on this project, won't depend on flink-json. -->
-			<optional>true</optional>
-		</dependency>
-
 		<!-- test dependencies -->
 
-		<dependency>
-			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-avro</artifactId>
-			<version>${project.version}</version>
-			<scope>test</scope>
-			<type>test-jar</type>
-		</dependency>
-
-		<dependency>
-			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-json</artifactId>
-			<version>${project.version}</version>
-			<type>test-jar</type>
```

Comment:
@yanghua Do we need to generate test jars for `flink-json` and `flink-avro` anymore?

Reply:
@twalthr The specific test cases have been removed and there is no error when building by maven. Did I miss anything else?

Reply:
In order to share test classes across modules a module must build `test-jar`s. If the test jar of `flink-json` and `flink-avro` are not necessary anymore, we can remove the jar building section from those `pom.xml`.